Lib/tkinter/filedialog.py

```diff
@@ -11,14 +11,19 @@
 directory dialogue available in Tk 8.3 and newer.
 These interfaces were written by Fredrik Lundh, May 1997.
 """
+__all__ = ["FileDialog", "LoadFileDialog", "SaveFileDialog", "Open", "SaveAs",
+           "Directory", "askopenfilename", "asksaveasfilename", "askopenfilenames",
```

Comment:
This line is longer then PEP 8 limit. I would format the list like:
```
__all__ = ["FileDialog", "LoadFileDialog", "SaveFileDialog",
           "Open", "SaveAs", "Directory",
           "askopenfilename", "asksaveasfilename", "askopenfilenames",
           "askopenfile", "askopenfiles", "asksaveasfile", "askdirectory"]
```

Reply:
Added the 79-character line length to my IDE's configuration for cPython